src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -170,9 +168,9 @@ describe('Results View', () => {
         results: [testData[0]],
       });
 
-    window.alert = jest.fn();
+    jest.spyOn(window, 'alert').mockImplementation(() => {});
     const mockedWindowAlert = window.alert as jest.Mock;
-    window.open = jest.fn();
+    jest.spyOn(window, 'open').mockImplementation();
```

Comment:
Interesting: the Jest documentation doesn't include this option, of calling `mockImplementation` without a parameter. This seems to work though... I'm a bit worried that this might fail in the future, but the risk is fairly small so I'm fine with keeping it this way.

Can you just make it the same for `window.alert` above? For consistency?